.gitignore

```diff
@@ -1,3 +1,6 @@
-custom/config.yml
-custom/people.opml
-custom/people.opml.bak
+/admin/inc/pwd.inc.php
+/cache
+/custom/config.yml
+/custom/people.opml
+/custom/people.opml.bak
+/nbproject/private/
```

Comment:
It could be removed, but .gitignore files often contain lists of rather random files that should be ignored... just makes things neater for people using various IDEs.